### PR TITLE
Back button refactor

### PR DESCRIPTION
**Pull request by kapoliub**

### CODE REVIEW

**Review by vasylshpak — APPROVED** [review]
Great job 🔥

**Comment by Iryna-Bzdyr** [comment]
Plese change it here to 
image

**Review by Iryna-Bzdyr — COMMENTED** [review]
Please change it here to
image

**Review by Iryna-Bzdyr — APPROVED** [review]

File: src/components/buttons/back-button/back-button.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const BackButton = ({ type, variant, color, ...props }) => {
+  const { GO_BACK_TITLE } = buttonTitles;
+  const history = createBrowserHistory();
+  return (
+    <Button
+      type={type}
+      color={color}
+      variant={variant}
+      onClick={() => history.goBack()}
+      {...props}
```
Comment (moran711) [inline src/components/buttons/back-button/back-button.js:16]: Its better to set `data-cy='back-btn'` here.

Comment (kapoliub) [reply src/components/buttons/back-button/back-button.js:17]: fixed

**Review by moran711 — APPROVED** [review]

**Pull request merged**